util/sys/linux/sys_test.go

```diff
@@ -10,25 +10,18 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package linux_test
```

Comment:
I think it is better to keep the test binary a separated package whenever possible/feasible.